bin/rosie-create

```diff
@@ -46,6 +46,10 @@
 #     --prefix=PREFIX
 #         Specify the prefix (i.e. the suite repository) to use.
 #         The default behaviour is to use the site default.
+#     --project=PROJECT
+#         Specify a project to check/query any available metadata when
+#         using rosie create.
+#         The default behaviour is to use no project and metadata.
```

Comment:
No need to say *when using rosie create*.

Need to update `doc/rose-command.html`.